README.md

```diff
@@ -2,6 +2,9 @@ Title: endless-runner-html5-game<br>
 Author: Steven Lambert<br>
 Website: http://blog.sklambert.com/endless-runner-html5-game/<br>
 Play Game: https://tutsplus.github.io/kandi-runner/<br>
+
+[![endless-runner](https://raw.githubusercontent.com/ashishsiot/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```

Comment:
With New Header Name

Reply:
Thanks for changing the image alt. 

Due to security reasons the image URL must be from this repo. Otherwise I don't control the actual image and (not that I'm saying you would) you could easily change the image on your end without my knowledge and it would then show in this repository. 

Since you've uploaded the image with the pull request, we just need change the url to this repo and it should work after merge.

```suggestion
[![endless-runner](https://raw.githubusercontent.com/straker/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```